### ISSUE TITLE
EasySnippetTest.py

### ISSUE THREAD

**Issue by jmhannon** [ASCOMInitiative/alpyca#14]
Just looking around and noticed that the file EasySnippetTest.py is empty.

**Comment by BobDenny** [comment]
Yeah this is an ancient file I was using to test an extension that would put a canned header (EasySnippetModuleHeader.py) into new files. I gave up as too much trouble for too little work saved. I'll delete the files. There may be some other trash files too ha ha.
